main.gd

```diff
@@ -0,0 +1,57 @@
+extends Node2D
+
+var interactive_object = preload("res://scenes/interactive_object.tscn")
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	GameManager.connect("start_new_round", handle_new_round)
+	GameManager.connect("game_finish", handle_game_finish)	
```

Comment:
Okay cool! This is a good pattern, essentially we can establish a listener through code this way without having both of the scenes in the same graph.